packages/web-pkg/src/components/AppTemplates/PartialViews/LoadingScreen.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="text-center flex justify-center items-center h-full">
     <oc-spinner size="xlarge" />
-    <p v-translate class="oc-invisible">Loading app</p>
+    <p v-translate class="invisible">Loading app</p>
```

Comment:
No `v-translate`, could you please fix this to `v-text="$gettext('Loading app')"` on the way? Also: why is it invisible? Shouldn't it be `sr-only` (I know, was wrong before as well)?